nix/default.nix

```diff
@@ -27,27 +24,33 @@ let
     ++ iohkNix.overlays.iohkNix
     # our own overlays:
     ++ [
-      (pkgs: _: with pkgs; {
-
-        # commonLib: mix pkgs.lib with iohk-nix utils and our own:
-        commonLib = lib // iohkNix
-        // import ./util.nix { inherit lib haskell-nix; }
-        # also expose our sources and overlays
-        // { inherit overlays sources; };
-      })
+      # Modifications to derivations from nixpkgs
       (import ./overlays/nixpkgs-overrides.nix)
       # This contains musl-specific stuff, but it's all guarded by appropriate host-platform
       # checks, so we can include it unconditionally
       (import ./overlays/musl.nix)
       # add pre-commit-hooks which isn't available in 20.03
       (import ./overlays/pre-commit-hooks.nix)
+      # add easyPS
+      (import ./overlays/easy-purescript.nix)
+      # sphinx haddock support
+      (import ./overlays/sphinx-haddock.nix)
+      # nodejs headers  (needed for purescript builds)
+      (import ./overlays/nodejs.nix)
```

Comment:
I think maybe all of these should just be in `_pkgs`?

Reply:
I guess my reasoning is that I consider those as "niv dependencies" and i'd to introduce and apply all of them in the same place - even though i realize that you could argue that  the sphinx stuff is local tool for development and should thus be in pkgsLocal...